core/src/wallet/ethereum/ERC20/ERC20LikeAccount.cpp

```diff
@@ -126,6 +190,47 @@ namespace ledger {
             return result;
         }
 
+        std::vector<std::shared_ptr<api::ERC20LikeOperation>>
+        ERC20LikeAccount::getSortedOperationsRange(
+            const std::chrono::system_clock::time_point& endDate
+        ) {
+            auto localAccount = _account.lock();
+            soci::session sql (localAccount->getWallet()->getDatabase()->getPool());
+            soci::rowset<soci::row> rows = (sql.prepare << "SELECT op.uid, op.ethereum_operation_uid, op.account_uid,"
```

Comment:
[OPT] we only need type and value, no need to ask for all the fields.

Reply:
What do you mean?

Reply:
SELECT op.date, op.value, op.type FROM erc20_operations ....

Reply:
If you do that, you cannot return a correct and coherent `ERC20Operation`.